flag.go

```diff
@@ -144,13 +142,14 @@ func (f FeatureFlag) evaluateExplain(user User, store FeatureStore, events *[]Fe
 			}
 		} else {
 			failedPrereq = &prereq
```

Comment:
Does this still send all FeatureRequestEvents for all prerequisites, even if the prereq condition is not met?